docs/docs/aleph_zero/sdk.md

```diff
@@ -0,0 +1,139 @@
+---
+title: SDK
+
+slug: /aleph_zero/sdk
+---
+
+SDK (Software Development Kit) can be used to interact with our contract programatically.
+
+## Build
+
+### Installation
+
+Clone repository
+
+```bash
+git clone git@github.com:invariant-labs/protocol-a0.git
+```
+
+Install NPM packages & build WASM from existing contract
+
+```bash
+cd sdk
+npm i
+npm run wasm:build
+npm run build
+```
+
+### Testing
+
+Install substrate-contracts-node
+
+```bash
+cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git
+```
+
+Run test script
+
+```bash
+./tests.sh
+```
+
+## Overview
+
+Our SDK consists of 3 separate contracts: our DEX contract (Invariant), token contarct (PSP22), and contract that can be used to wrap native currency (Wrapped AZERO) and various helper functions. You can deploy your own or use existing address of any of those contracts. Network enum allows you to choose network you want to perform actions to. In order to make a contract call you have to use methods of classes of loaded or deployed contracts. First parameter will always be an account you want to call from and rest of parameters will be entrypoint parameters. Invariant class has additional methods that allows you to attach event listeners so you can run your own code based on contract activity.
+
+## Usage
+
+### Starting off
+
+```javascript
+const main = async () => {
+  const api = await initPolkadotApi(Network.Main) // initialize api, use enum to specify the network
+
+  const keyring = new Keyring({ type: 'sr25519' })
+  const account = await keyring.addFromUri('//Alice') // initialize account (we used dummy account, use your own)
+}
+
+main()
+```
+
+### Initialize DEX and tokens
+
+:::note
+Notice how we use "n" at the end of every number. "n" indicates that specified value is a BigInt, number with higher precision. We use it almost everywhere in our SDK.  
+:::
+
+:::warning
+Tokens are sorted alphabetically when pool key is created, so make sure that you swap tokens in correct direction.
```

Comment:
You can add a link in similar cases pointing to a broader description (somewhere inside contract description, there for sure be description)